### PR TITLE
Add effect for Spirit Guide Form

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/feats/spirit-guide-form.json [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
+            "value": "<p><strong>Frequency</strong> once per day</p>\n<hr />\n<p>You fuse with your spirit guide, becoming a hybrid being with enhanced physical and spiritual power. This effect lasts 1 minute; you can end it early as a free action. During this time, you can't use any other actions granted by this archetype. You gain the following statistics and abilities, regardless of your type of spirit guide.</p>\n<ul>\n<li>You become incorporeal.</li>\n<li>You gain resistance 10 to all damage (except force, ghost touch, or vitality; double resistance vs. non-magical).</li>\n<li>Darkvision.</li>\n<li>An unarmed melee attack of the same type as your spirit guide (either claws or jaws), which is the only attack you can Strike with. You're trained with this attack. Your attack modifier is +29 or your normal unarmed attack modifier, whichever is higher. You use the listed statistics for the unarmed attack type:\n<ul>\n<li><strong>Claws</strong> (agile, backstabber, finesse) <strong>Damage</strong> 3d6+5 slashing plus 1d6 force</li>\n<li><strong>Jaws</strong> (forceful) <strong>Damage</strong> 3d10+5 piercing plus 1d6 force.</li>\n</ul>\n</li>\n</ul>"
```
Comment (SpartanCPA) [inline packs/feats/spirit-guide-form.json:14]: The form should give only one strike based on a choice.

Comment (simonward) [reply packs/feats/spirit-guide-form.json:14]: True, but there isn't a way to currently handle this.

Comment (TikaelSol) [reply packs/feats/spirit-guide-form.json:14]: Could we maybe get fancy with SpecialStatistic? We haven't mucked with that much yet.

Comment (simonward) [reply packs/feats/spirit-guide-form.json:14]: I'm not sure how to do this. Documentation and examples for SpecialStatistic are pretty sparse.

Comment (TikaelSol) [reply packs/feats/spirit-guide-form.json:14]: yeah, it's on my list to remedy before i do the RE videos

Comment (simonward) [reply packs/feats/spirit-guide-form.json]: Now uses the new predicate support for BattleForm Strikes.

**Pull request merged**